### PR TITLE
DMOJ Rich Presence Update

### PR DESCRIPTION
**Pull request by vidhanio**
I have completely rewritten the presence and added many new states and images, as well as buttons where they are useful.

Screenshots:

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Run prettier and remove some useless empty lines

**Comment by vidhanlol** [comment]
done!

**Review by Slowlife01 — DISMISSED** [review]

File: websites/D/DMOJ/dist/metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "$schema": "https://schemas.premid.app/metadata/1.3",
```
Comment (SlowLife1661) [inline websites/D/DMOJ/dist/metadata.json:2]: 1.4 is now the latest.

Comment (vidhanlol) [reply websites/D/DMOJ/dist/metadata.json]: updated! :)

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**